packages/console/src/components/MultiTextInputField/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { conditional } from '@silverhand/essentials';
+
+import type { Props as FormFieldProps } from '@/components/FormField';
+import FormField from '@/components/FormField';
+import type { Props as MultiTextInputProps } from '@/components/MultiTextInput';
+
+import MultiTextInput from '../MultiTextInput';
+import * as styles from './index.module.scss';
+
+type Props = MultiTextInputProps &
+  Pick<FormFieldProps, 'isRequired' | 'tooltip'> & {
+    formFieldClassName?: FormFieldProps['className'];
+  };
+
+const MultiTextInputField = ({
+  title,
+  isRequired,
+  tooltip,
+  formFieldClassName,
+  value,
+  ...rest
+}: Props) => (
+  <FormField
+    title={title}
+    isRequired={isRequired}
+    tooltip={tooltip}
+    className={formFieldClassName}
+    headlineClassName={conditional(value && value.length > 1 && styles.headlineWithMultiInputs)}
```

Comment:
nit: `value?.length`

Reply:
We can't.
<img width="571" alt="image" src="https://user-images.githubusercontent.com/10806653/204508757-8c5b78b4-305e-4c36-a331-efa51f4470c6.png">